nsasm/parse.cc

```diff
@@ -398,6 +398,7 @@ ErrorOr<Directive> ParseDirective(TokenSpan* pos) {
       return std::move(directive);
     }
   }
+  return Error("unknown directive_type: %d", directive_type);
```

Comment:
This is sad; it means that the compiler will no longer protect against new entries being added to the enum.

Reply:
Yeah, this is a pessimisation for the compiler that we're actively developing this on.